lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "3.15.0",
+  "lerna": "5.4.0",
```

Comment:
This might produce some issues since latest lerna can crash in the pipeline. We would need to make some kind of proper upgrade to the latest version sometime I guess. 
Latest version that I remember would work was 4.0.0 form [this PR](https://github.com/SAP/luigi/pull/3322/files)